tests/libvmi/network.go

```diff
@@ -25,7 +25,10 @@ import (
 	"kubevirt.io/kubevirt/tests/libnet"
 )
 
-const DefaultInterfaceName = "default"
+const (
+	DefaultInterfaceName = "default"
+	PasstBindingName     = "passt"
```

Comment:
please move this with the plugin helpers to a dedicated file.
Perhaps `netbindingplugin.go`.

Reply:
Why? The file contains helpers for SRIOV, Slirp, Masquerade and more. Why passt deserves a separate file?

Reply:
I think in the prev version, it had many changes added.
It is also a plugin, so it makes sense not to overload this file with more.

As it looks now, I guess we do not yet need to do this.
But when you'll add macvtap for example, it will be much clearer it fits a new file.